TLSharp.Core/TelegramClient.cs

```diff
@@ -28,8 +29,10 @@ public class TelegramClient : IDisposable
         private int _apiId = 0;
         private Session _session;
         private List<TLDcOption> dcOptions;
+        private TcpClientHandler _handler;
 
-        public TelegramClient(int apiId, string apiHash, ISessionStore store = null, string sessionUserId = "session")
+        public TelegramClient(int apiId, string apiHash, ISessionStore store = null, string sessionUserId = "session",
+            TcpClientHandler handler = null)
```

Comment:
If you break this line in two, it feels better to place all mandatory parameters in one line, and all optional parameters in the next line, you mind doing that?